utils/email_backends.py

```
@@ -0,0 +1,29 @@
+from django.core.mail.backends.base import BaseEmailBackend
+from django.conf import settings
+from django.core.mail import EmailMessage
```

Ruff (0.8.2)

`django.core.mail.EmailMessage` imported but unused

Remove unused import: `django.core.mail.EmailMessage`

(F401)